src/components/application_manager/test/commands/mobile/set_audio_streaming_indicator_test.cc

```diff
@@ -494,18 +500,14 @@ TEST_F(SetAudioStreamingIndicatorRequestTest,
 
   ApplicationSharedPtr mock_app_empty;
   EXPECT_CALL(app_mngr_, application(_))
-      .Times(2)
+      .Times(1)
       .WillRepeatedly(Return(mock_app_empty));
 
   EXPECT_CALL(*mock_app_, RemoveIndicatorWaitForResponse(_)).Times(0);
   EXPECT_CALL(
       mock_message_helper_,
       CreateNegativeResponse(_, _, _, am::mobile_api::Result::GENERIC_ERROR))
       .WillOnce(Return(msg_mobile_response));
-  EXPECT_CALL(
-      app_mngr_,
-      ManageMobileCommand(_, am::commands::Command::CommandOrigin::ORIGIN_SDL))
-      .WillOnce(DoAll(SaveArg<0>(&msg_mobile_response), Return(true)));
 
   command->onTimeOut();
 
```

Comment:
@pold500 Title of this commit is too big, please reword it to make it more compact and instead extend the commit description.